contributing.md

```diff
@@ -0,0 +1,29 @@
+## Contributing
```

Comment:
See this PR for example: https://github.com/rudi-c/the-waterloo-blogger/pull/8

The person didn't put the new blog in alphabetical order, so I just did it for them rather than make a round-trip. The changes I made are still visible as a commit.

When you go in files changed, you can edit individual files. I'm not quite sure if there's an easy way to do it on your own machine (I imagine it requires setting up your .git/config to pull in pull requests as branches) but for simple text files that's probably not necessary.